ansible/roles/logger-service/templates/logback.xml

```diff
@@ -6,7 +6,7 @@
     <conversionRule conversionWord="wex" converterClass="org.springframework.boot.logging.logback.WhitespaceThrowableProxyConverter" />
 
     <variable name="appName" value="logger-service"/>
-    <variable name="LOG_DIR" value="/var/log/{{ tomcat | default('tomcat9')}}" />
+    <variable name="LOG_DIR" value="{{ jar_log_path | default('/var/log/tomcat9')}}" />
```

Comment:
IMHO this should retain the tomcat version in the log path for the default case.
ie `{{ jar_log_path | default('/var/log/'  + tomcat)}}` or however you'd express that in ansible

Reply:
Fixed